messaginginapp/src/test/java/io/customer/messaginginapp/ModuleMessagingInAppTest.kt

```diff
@@ -43,14 +45,15 @@ internal class ModuleMessagingInAppTest : JUnitTest() {
                 diGraph {
                     sdk {
                         overrideDependency<ScopeProvider>(testScopeProviderStub)
+                        val spykEventBus = spyk(eventBus)
+                        overrideDependency<EventBus>(spykEventBus)
```

Comment:
Could be simplified?

```suggestion
                        overrideDependency<EventBus>(spyk(eventBus))
```